docs/Form-Question-Descriptions.md

```diff
@@ -0,0 +1,47 @@
+**2. Form Question Descriptions**
+    
+A. Researcher Name: Name of the researcher inputting the dataset in Read the Doc
+
+B. Project Name: Choose the name of the project the researcher is using the dataset for.
+
+C. Dataset Name: Name of the dataset created by the publisher
+
+D. Description: Indicate what the dataset is and why the dataset is used for the specified project. Indicate any provinces, states or municipalities the dataset includes if applicable.
+
+E. Version: Version of the dataset if applicable
+
+F. Private or Public: Indicate whether the dataset is publicly available or if it is private and requires special requests to access
+
+G. Region: Indicate the lowest regional level available within the dataset. Region levels are described as World > Continent > Country
+
+H. Time Horizon: Indicate the dataset timespan from the beginning of the acquired data and to the last available data point. Only year is required to be entered. If the dataset is continuously updating, add the time of the first available data point and present in the to side. If the dataset does not include a time horizon, write when the dataset was created
+
+I. Spatial resolution: If land size is included in the dataset, include the spatial resolution within the specified ranges. Ranges are in km2. If land size is in other formats, convert to km2
+
+J. Temporal Resolution: If temporal resolution is included in the dataset include the resolution(s) included within the data.
+
+K. Link to Access: Include the access links or methodology for how to access the datasets, as well as any requirements or contacts to access and use the dataset
+
+L. Citation Requirements: Write down any publisher specified citation requirements
+
+M. Additional Information: Indicate any additional information not included here that can be useful for the use of this dataset
+
+N.  Taglist Descriptions: The following are the standarised list section and a check mark on the drop down list can be used for any and/or all items on the list
```

Comment:
Do we think we need to say what the heading for the tags are, or do you think that's self-explanatory?

Reply:
I think most of them are self explanatory but I can add that in.

Reply:
I think adding explanation for heading tags are def useful. On this note we should rework the "area of focus" tag. While defining you will probably think of something more descriptive for its header.

Reply:
Maybe single words as well for easier tagging? (dtype instead of Data Type?) Or do you think a tag like `Data-Type:Geospatial` is fine?

Reply:
Single words is a good idea. In that case, make sure to define them in the definitions you are adding, @malshebo .

Reply:
Can do!